draft-ietf-tls-tls13.md

```diff
@@ -2183,8 +2183,10 @@ hidden by TCP.
 
 A small allowance for errors in clocks and variations in measurements
 is advisable.  However, any allowance also increases the opportunity
-for replay.  In this case, it is better to reject early data than to
-risk greater exposure to replay attacks.
+for replay.  In this case, it is better to reject early data and fall back
+to a full 1-RTT handshake than to risk greater exposure to replay attacks,
+so in common network topologies, a small allowance on the order of ten
+seconds is reasonable.
```

Comment:
@martinthomson The data over a period of a week. The results hold even over a larger time window. 

The assumption of the ticket age method is that the clock skew remains the same over time. I can't tell you exactly how often clocks drift. However I can say how much 0rtt we would lose if we set the client skew to 10s, or any X. So when we receive a 0RTT attempt, if we enforced 10s as the ticket age skew, then how much 0rtt would we end up rejecting from client that were within a reasonable absolute client time. Does that clear things up?

The time window will be server specific. Any number we write down is not enforceable in the protocol, so we're just writing down a recommended number. Servers can tune this as they want based on their tolerance and different servers have different amounts of 0rtt traffic they can lose. I'm just noting at what level we've chosen to draw that line which is 0.1%. You might be fine with 0.3%. We're also re-evaluating that line with some other experiments.

About to get on a flight so will be AWOL for a few hours.